src/lib.rs

```diff
@@ -408,61 +408,155 @@
 //!
 //! ## Help messages
 //!
-//! Help messages for the whole binary or individual arguments can be
-//! specified using the `about` attribute on the struct and the `help`
-//! attribute on the field, as we've already seen. For convenience,
-//! they can also be specified using doc comments. For example:
+//! In `clap`, help messages for the whole binary or individual arguments
+//! can be specified via [`App::about`], [`App::long_about`], [`Arg::help`], and
+//! [`Arg::long_help`]. `long_*` variants are used when user calls the program with
+//! `--help` and "short" variants are used with `-h` flag. In `structopt`,
+//! you can use them via [raw methods](#raw-methods), for example:
 //!
 //! ```
 //! # use structopt::StructOpt;
 //!
 //! #[derive(StructOpt)]
-//! /// The help message that will be displayed when passing `--help`.
+//! #[structopt(about = "I am a program and I work, just pass `-h`")]
 //! struct Foo {
-//!   #[structopt(short)]
-//!   /// The description for the arg that will be displayed when passing `--help`.
+//!   #[structopt(short, help = "Pass `-h` and you'll see me!")]
 //!   bar: String
 //! }
-//! # fn main() {}
 //! ```
 //!
-//! If it is necessary or wanted to provide a more complex help message then the
-//! previous used ones, it could still be a good idea to distinguish between the
-//! actual help message a short summary. In this case `about` and `help` should
-//! only contain the short and concise form while the two additional arguments
-//! `long_about` and `long_help` can be used to store a descriptive and more in
-//! depth message.
-//!
-//! If both - the short and the long version of the argument - are present,
-//! the user can later chose between the short summary (`-h`) and the long
-//! descriptive version (`--help`) of the help message. Also in case
-//! of subcommands the short help message will automatically be used for the
-//! command description inside the parents help message and the long version
-//! as command description if help is requested on the actual subcommand.
-//!
-//! This feature can also be used with doc comments instead of arguments through
-//! proper comment formatting. To be activated it requires, that the first line
-//! of the comment is separated from the rest of the comment through an empty line.
-//! In this case the first line is used as summary and the whole comment represents
-//! the long descriptive message.
+//! For convenience, doc comments can be used instead of raw methods
+//! (this example works exactly as one above):
```

Comment:
I thought "one" can be considered a replacement for an article of sorts?

Reply:
You must have both a noun and and article in this context. If "one" is acting as an article, then you're missing a noun. If "one" is acting as a noun, then you're missing an article... and using "one" as a noun in this context wouldn't feel natural, because it would mean "one of multiple" but you only gave one prior example.

My suggestion of "the one above" was equivalent to "the example above".